Servers/UI/OJS.Servers.Ui/ClientApp/src/layout/nav/PageNav.tsx

```diff
@@ -37,8 +37,7 @@ const PageNav = () => {
         const type = isPrimary
             ? LinkButtonType.primary
             : LinkButtonType.secondary;
-
-        const btnClassName = styles.btn;
+        const { btn: btnClassName } = styles;
```

Comment:
This should not be necessary. I am not sure that it works, actually.

Reply:
I reverted the change and suppressed the rule.